vignettes/getting-started.Rmd

```diff
@@ -50,22 +50,23 @@ The _first time_ you use `rbabylon` on a system or disk, you must first install
 
 To reiterate, there are two relevant paths for this section.
 
-* `BBI_EXE_PATH` -- the path the `babylon` executable file on your system.
+* `BBI_EXE_PATH` -- the path to the `babylon` executable file on your system.
 * `MODEL_DIR` -- the root modeling directory where you will be running models with `rbabylon`
 
 ```{r define dirs}
-BBI_EXE_PATH <- "/data/apps/bbi" # this should be an absolute path
+BBI_EXE_PATH <- read_bbi_path()
```

Comment:
I think we should actually have two different blocks here:
1. this new one, but set to `include = FALSE`
2. the old one, but set to `eval = FALSE`

My reasoning is that we're showing the user how to set this and we _don't_ actually want them to set it with `read_bbi_path()`, we want them to set it explicitly (ideally in their .Rprofile, as we describe later).

Reply:
Good point, I've made this change.